containers/demo/ctia/config/ctia.properties

```diff
@@ -1,14 +1,5 @@
 ctia.auth.type=allow-all
-ctia.http.port=3000
-ctia.http.min-threads=10
-ctia.http.max-threads=100
-ctia.nrepl.port=3001
-ctia.nrepl.enabled=true
-ctia.store.type=es
-ctia.store.sql.db.classname=org.h2.Driver
-ctia.store.sql.db.subprotocol=h2
-ctia.store.sql.db.subname=/tmp/ctia-h2-db;DATABASE_TO_UPPER=false
-ctia.store.sql.db.delimiters=
+ctia.store.redis.uri=redis://redis:6379
```

Comment:
I take it that these are names that refer to the redis and the elastic-services hosts?